src/main/java/duke/exceptions/DukeIDKException.java

```diff
@@ -0,0 +1,23 @@
+package duke.exceptions;
+
+public class DukeIDKException extends DukeException{
```

Comment:
Should there be a space between `DukeException` and the opening bracket `{`? 
I noticed the same issue in several other places too.